src/ctia/entity/feed.clj

```diff
@@ -218,226 +218,251 @@
      :responses {404 {:schema s/Str}
                  401 {:schema s/Str}}
      :query-params [s :- (describe s/Str "The feed share token")]
-     (let [{:keys [output]
-            :as feed} (fetch-feed id s services)]
+     :header-params [{x-search_after :- (s/maybe s/Str) "nil"}
+                     {x-limit :- (s/maybe s/Str) "nil"}]
```

Comment:
Could you provide the same [pagination interface](https://github.com/threatgrid/ctia/blob/master/src/ctia/http/routes/common.clj#L51) we already expose in the other API endpoints? They use `limit`, `offset` and `search_after` as query parameters.

Reply:
done